test/Project.toml

```diff
@@ -11,6 +11,7 @@ Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
You should add Revsie to your global environment (the one you get by starting Julia without `--project`), not ClimaCore (Julia "stacks" environments, so anything in the global environment is accessible elsewhere)

Reply:
Agreed. This was a slip! I didn't mean for this to be pushed